internal/mode/static/manager.go

```diff
@@ -402,12 +403,14 @@ func registerControllers(
 		},
 		{
 			objectType: &apiv1.Service{},
+			name:       "user-service",
```

Comment:
Let's add a comment here and on line 413 explaining why we need to define the name for the service controllers